plugin/evm/vm.go

```diff
@@ -545,6 +545,8 @@ func (vm *VM) Initialize(
 	vm.ethConfig.SnapshotDelayInit = vm.stateSyncEnabled(lastAcceptedHeight)
 	vm.ethConfig.SnapshotWait = vm.config.SnapshotWait
 	vm.ethConfig.SnapshotVerify = vm.config.SnapshotVerify
+	vm.ethConfig.HistoricalProofs = vm.config.HistoricalProofs
```

Comment:
We should highlight this change in release notes as API providers for getProof may want to update their settings here.